server/src/main/resources/reference.conf

```diff
@@ -27,6 +27,11 @@ rest-stats {
     bind-hostname = "0.0.0.0"
 }
 
+queue-persister {
```

Comment:
I think a better config name would be `queues-storage` (as a complement to `queues`), with an `enabled` flag. So sth like this:
```
queues-storage {
  enabled = false
  path = ""
}
```